evm/p2p/kademlia.py

```diff
@@ -111,32 +120,32 @@ def __hash__(self):
 
 
 @total_ordering
-class KBucket:
+class KBucket(Sized):
```

Comment:
*nitpick*

This also qualifies to be marked as `collections.abc.Container`.  If we were to add an `__iter__` we could just use `collections.abc.Set` which I think is actually appropriate.  We have at least one place that I could find where we iterate over the nodes in a bucket.

Reply:
So, it turns out that if I use a `collections.abc` base class here I also need to change the type annotations to match those of that base class: i.e. in `__contains__()` I need to use `object` instead of `Node` as the argument's type. I think I prefer to use the appropriate type, so not adding the base class here